Resources/config/services.xml

```diff
@@ -19,9 +21,15 @@
     </parameters>
```

Comment:
@weaverryan Totally agreed. As long as I worked on different files I saw things that need to be changed, such as the documentation that needs more clean examples with namespaces, following best practices. It itches me to change them, but as you said, it's out of the scope of this PR. So, I'll propose this changes in a next one. Thank you for the review!